vite.config.ts

```diff
@@ -1,7 +1,28 @@
-import { defineConfig } from 'vite'
-import react from '@vitejs/plugin-react'
+/// <reference types="vitest"/>
+/// <reference types="vite/client"/>
+
+import { defineConfig } from "vite";
+import react from "@vitejs/plugin-react";
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-})
+  test: {
+    globals: true,
+    environment: "jsdom",
+    setupFiles: [`./src/setupTest.ts`],
+    coverage: {
+      reporter: ["text", "html", "lcov", "json-summary", "text-summary"],
+      exclude: [
+        "node_modules/",
+        ".eslintrc.cjs",
+        "coverage",
+        "postcss.config.js",
+        "tailwind.config.js ",
+        "src/pages/Users.tsx",
+        "src/pages/OtherUsers.tsx ",
+        "src/redux",
```

Comment:
why you have ignored this pages @Hakizimana-Clement ?
Users.tsx
OtherUsers.tsx

Reply:
we need one more test script for monitoring our test in ui.
`"test:ui": "vitest --ui"`